### ISSUE TITLE
shellcheck file ignored

### ISSUE THREAD

**Issue by Xat59** [github/super-linter#2577]
## Describe the bug

I have a repo that uses `super-linter` in a github workflow : 

```
name: Shellcheck

on:
  pull_request:
    branches:
      - 'main'
      - 'master'

env:
  REPO_DEFAULT_BRANCH: master

jobs:
  quality:
    runs-on: ubuntu-latest
    steps:
      - name: Linter
        uses: github/super-linter/slim@v4
        env:
          DEFAULT_BRANCH: ${{ env.REPO_DEFAULT_BRANCH }}
          VALIDATE_ALL_CODEBASE: "false"
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          VALIDATE_TERRAFORM_TERRASCAN: "false"
          YAML_ERROR_ON_WARNING: "false"
```

I need to exclude specific rule from being checked with shellcheck.
Regarding the documentation provided here : https://github.com/github/super-linter/blob/main/docs/disabling-linters.md , we need to create a configuration file per linter within the `.github/linters/` folder. Each linter configuration is based on [the template file](https://github.com/github/super-linter/blob/main/TEMPLATES/.shellcheckrc) (in my case for shellcheck).

So I've created a file `.github/linters/.shellcheckrc` with this content : 

```
source-path=SCRIPTDIR
disable=SC2043
```

However, the shellcheck linter still warns us about the `SC2043` rule.

## Expected behavior

My github workflow should not warn about the `SC2043` rule when linting with shellcheck.

## Steps to Reproduce

1. Create a github workflow with `github/super-linter/slim@v4` like provided above.
2. Create a shell script that triggers a warning with the shellcheck linter
3. Create the `.github/linters/.shellcheckrc` file with the content provided in the description above (note you have to adapt the `disable` line with your rule code. 
4. Execute the github workflow

## Logs

Here are logs from my github workflow : 

```
2022-03-03 10:59:40 [INFO]   ---------------------------------
2022-03-03 10:59:40 [INFO]   ------ File list to check: ------
2022-03-03 10:59:40 [INFO]   ---------------------------------
2022-03-03 10:59:41 [INFO]   ----------------------------------------------
2022-03-03 10:59:41 [INFO]   Successfully gathered list of files...
2022-03-03 10:59:42 [INFO]   
2022-03-03 10:59:42 [INFO]   ----------------------------------------------
2022-03-03 10:59:42 [INFO]   ----------------------------------------------
2022-03-03 10:59:42 [INFO]   Linting [GITHUB_ACTIONS] files...
2022-03-03 10:59:42 [INFO]   ----------------------------------------------
2022-03-03 10:59:42 [INFO]   ----------------------------------------------
2022-03-03 10:59:42 [INFO]   ---------------------------
2022-03-03 10:59:42 [INFO]   File:[/github/workspace/.github/workflows/terragrunt.yml]
2022-03-03 10:59:42 [ERROR]   Found errors in [actionlint] linter!
2022-03-03 10:59:42 [ERROR]   Error code: 1. Command output:
------
.github/workflows/terragrunt.yml:42:9: shellcheck reported issue in this script: SC2043:warning:1:13: This loop will only ever run once. Bad quoting or missing glob/expansion? [shellcheck]
   |
42 |         run: |
   |         ^~~~
------
```

## Additional context

I also tried to configure shellcheck at several locations but same behavior  : 

- `.github/linters/.shellcheckrc`
- `.github/linters/.shellcheck`
- `.shellcheckrc`
- `.shellcheck`

Can you confirm the location of the linters configuration file ? 
Can you confirm the shellcheck configuration file name ?

Thanks for your help.
Regards,

**Comment by BretFisher** [comment]
That's an error in the `actionlint` linter, not in super-linter itself, as indicated by the line:

`2022-03-03 10:59:42 [ERROR]   Found errors in [actionlint] linter!`

You should look at the [actionlint](https://github.com/rhysd/actionlint) documentation and see how you tell it to ignore the errors it gets when running spellcheck internally. I do it by adding a disable line inside the `run:` of a workflow:
```yaml
        run: |
          # shellcheck disable=2043
```

**Comment by ferrarimarco** [comment]
Correct. Apparently, [actionlint explicitly disables loading a `.shellcheckrc` file](https://github.com/rhysd/actionlint/blob/8e5df1efeca760695f3f5e48bf18d89f24ac19b7/rule_shellcheck.go#L169).

If you're interested in this use case, it may be the case to open a request in the actionlint repo.
